dts/bindings/mbox/ti,omap-mailbox.yaml

```
@@ -0,0 +1,23 @@
+# Copyright 2024 Texas Instruments Incorporated.
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI OMAP MAILBOX
+
+compatible: "ti,omap-mailbox"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  usr_id:
```

I think dashes are more often used in device trees as separator like `interrupt-cells`
```suggestion
  usr-id:
```